src/app/blog/page.jsx

```diff
@@ -0,0 +1,150 @@
+"use client";
+import Card from '@/components/blogs/Card'
+import SearchAndFilter from '@/components/blogs/SearchAndFilter';
+import Blog_Modal from '@/components/blogs/Blog_Modal'
+import {Blogs} from '@/lib/blogStaticData'
+import React, { useEffect, useState } from 'react';
+
+const page = () => {
+    const [modal,setModal]=useState(false)
```

Comment:
Have separate page for each blog, rather than a modal which opens here itself. That way we can share individual blog links, and also more readable.